engine/pkg/orm/client_test.go

```diff
@@ -1270,6 +1267,33 @@ func TestLogicEpoch(t *testing.T) {
 	}
 }
 
+func TestContext(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.TODO(), 1*time.Second)
+	defer cancel()
+
+	cli, err := NewMockClient()
+	require.NoError(t, err)
+	defer cli.Close()
+
+	// test normal function
+	err = failpoint.Enable("github.com/pingcap/tiflow/engine/pkg/orm/initializedDelay", "sleep(2000)")
+	require.NoError(t, err)
+	err = cli.Initialize(ctx)
```

Comment:
We can use `InjectContext` to implement concurrent injection with failpoint, such as https://github.com/amyangfei/concurrent-failpoint